### DISCUSSION TITLE
Touchscreen support

### DISCUSSION THREAD

**Discussion by dcveteran** [OpenVoiceOS/ovos-buildroot#226] [Q&A] [answered]
Is there a config file change needed to enable a touchscreen? It's the official Raspberry Pi one. It worked at one time in one of the older buildroot versions, but I can't remember if I had to change something.

**Comment by builderjer** [comment, marked as answer]
It depends on how you install OVOS.  The raspOVOS image supports touch out of the box.  The buildroot image will also.  As for the installer, it expects you to have a system setup before hand, screen, sound, etc...  and it puts OVOS on top of that

**Reply by dcveteran** [reply]
I figured this one out. I discovered one of the small ribbon cables pulled free. User error on this one. Thanks.
